style.css

```diff
@@ -0,0 +1,679 @@
+}
+
+.section1 {
+  background: url('./images/mobile_intro_bg.png');
+  font-family: 'Roboto', sans-serif;
+  height: 80vh;
+}
+
+.sub-text {
+  display: flex;
+  flex-direction: column;
+  margin-left: 0.6rem;
+  text-align: left;
+  padding: 1rem;
+}
+
+.welcome {
+  font-family: 'Roboto', sans-serif;
+  color: #c03f3f;
+  font-weight: 400;
+  font-size: 16px;
+  padding: 1rem 0 0 0;
+  margin-bottom: 7px;
+  margin-top: 30px;
+}
+
+.main-title {
+  color: #c52222;
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  font-size: 4vh;
+  font-style: normal;
+  background-image: url('https://unsplash.it/800');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-clip: content-box;
+  -webkit-background-clip: text;
+}
+
+.saying .paragragh1 {
+  font-family: 'poppins', sans-serif;
+  margin-top: 0.8rem;
+  color: #272a32;
+  background-color: rgb(242, 239, 239);
+  border-radius: 1px;
+}
+
+.section1 footer {
+  margin-left: 0.6rem;
+  padding: 1rem;
+}
+
+.main-programs {
+  background: url('/images/pattern_bg.png');
+  background-repeat: repeat;
+  background-color: #272a32;
+  padding-bottom: 1rem;
+}
+
+.main-programs h2 {
+  text-align: center;
+  padding: 2rem 0 1rem 0;
+  color: #eee;
+}
+
+.main-programs .fast-line {
+  width: 4%;
+  margin: 0 auto;
+  border-color: #c52222;
+  margin-bottom: 1rem;
+}
+
+.program-parts {
+  display: flex;
+  flex-direction: column;
+  color: #eee;
+  gap: 10px;
+}
+
+.part {
+  width: 95%;
+  margin: 0 auto;
+  padding: 20px 7px;
+  background-color: rgba(255, 255, 255, 0.1);
+  display: flex;
+  align-items: center;
+}
+
+.part img {
+  width: 25%;
+  height: auto;
+  margin-right: 15px;
+}
+
+.part h3 {
+  color: #c52222;
+  width: 35%;
+  font-size: 0.9rem;
+  font-weight: bold;
+}
+
+.part p {
+  width: 65%;
+  font-size: 0.8rem;
+}
+
+.viewer {
+  margin: 1rem auto;
+  padding: 1rem 2rem;
+  background-color: #fa4503;
+  border-radius: 6px;
+  cursor: pointer;
+  font-size: 24px;
+  width: 300px;
+  color: #eee;
+}
+
+.artists {
+  display: flex;
+  flex-direction: column;
+  margin-top: 3rem;
+}
+
+.artists h2 {
+  text-align: center;
+}
+
+.artists .second-line {
+  width: 4%;
+  margin: 3px auto;
+  border-color: #fa0404;
+}
+
+.artist-info {
+  display: flex;
+  flex-direction: column;
+  margin-left: 0.5rem;
+}
+
+.artist-card {
+  display: flex;
+  margin: 1rem;
+}
+
+.artist-heading {
+  margin-top: 0.8rem;
+  margin-left: 0.5rem;
+}
+
+.paragragh {
+  margin-bottom: 0.2rem;
+  margin-left: 0.5rem;
+  font-size: 12px;
+  color: #c52222;
+}
+
+.artist-data {
+  font-size: 12px;
+  margin-left: 0.5rem;
+}
+
+.second-heading {
+  font-family: 'Roboto', sans-serif;
+  color: #c03f3f;
+  font-weight: 400;
+  font-size: 16px;
+  padding: 1rem 0 0 0;
+  margin: 10px auto;
+}
+
+.aboutme-content {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  font-family: 'Roboto', sans-serif;
+  background: url('./images/mobile_intro_bg.png');
+  height: 55vh;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly remove the `height: 55vh;` and the `padding: 1rem;` it will make your design much better.